docs/_data/portland-2022-schedule.yaml

```diff
@@ -0,0 +1,98 @@
+writing_day:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:30'
+    title: Team up and start working
+  - time: '12:00'
+    title: Lunch Break
+  - time: '12:30'
+    title: Conference Intro and Tour with the Welcome Wagon
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: Platform opens
+  - time: '8:30'
+    title: Welcome Wagon conference introduction and platform tour
+  - time: '9:00'
+    title: Write the Docs Team - Community update
+  - time: '9:30'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:30'
+    slug: TBD 1/2
+  - time: '10:00'
+    title: '5 minute break'
+  - time: '10:05'
+    slug: TBD 2/2
+  - time: '10:35'
+    title: '5 minute break'
+  - time: '10:40'
+    title: 'Two talk Q&A'
+  - time: '10:55'
+    title: 'Coffee break'
+  - time: '11:10'
+    slug: TBD 1/2
+  - time: '11:40'
+    title: '5 minute break'
+  - time: '11:45'
+    slug: TBD 2/2
+  - time: '12:15'
+    title: 'Two talk Q&A'
+  - time: '12:30'
+    title: 'Lunch Break'
+  - time: '1:30'
+    title: Lightning Talks (30 mins)
+  - time: '2:00'
+    slug: TBD 1/2
+  - time: '2:30'
+    title: '5 minute break'
+  - time: '2:35'
+    slug: TBD 2/2
+  - time: '3:05'
+    title: '5 minute break'
+  - time: '3:10'
+    title: 'Two talk Q&A'
+  - time: '3:25'
+    title: Day 1 wraps up
```

Comment:
Would it be easier to combine this with the final Q&A of the day? It's not a long update and usually done live, so it might help to consolidate things for people

Reply:
I reckon we're better off saying "The END" tbh, but don't feel very strongly

Reply:
I feel like the end of the day is always awkward. I don't think we have a great option, and people are always confused.